### PR TITLE
APM-299788 adjust one-liner and license

### PR DESCRIPTION
**Pull request by wojtasskorcz**

### CODE REVIEW

**Review by mmajcher — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ REQUIRE_VALID_CERTIFICATE=false
 Download the script and deploy the infrastructure:
 
 ```
-DT_AWS_LOG_FORWARDER=https://dynatraceawslogforwarder.s3.amazonaws.com && wget $DT_AWS_LOG_FORWARDER/preview/dynatrace-aws-logs.sh -O dynatrace-aws-logs.sh && chmod +x ./dynatrace-aws-logs.sh && wget $DT_AWS_LOG_FORWARDER/preview/lambda-aws-logs.zip -O lambda-aws-logs.zip && wget $DT_AWS_LOG_FORWARDER/preview/template-aws-logs.yaml -O template-aws-logs.yaml && ./dynatrace-aws-logs.sh deploy --target-url $TARGET_URL --target-api-token $TARGET_API_TOKEN --require-valid-certificate $REQUIRE_VALID_CERTIFICATE
+wget -q https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases/latest/download/dynatrace-aws-log-forwarder.zip && unzip -q dynatrace-aws-log-forwarder.zip && cd dynatrace-aws-log-forwarder && ./dynatrace-aws-logs.sh deploy --target-url $TARGET_URL --target-api-token $TARGET_API_TOKEN --require-valid-certificate $REQUIRE_VALID_CERTIFICATE
```
Comment (dt-be) [inline README.md:34]: I'm very much against quiet modes.  If I am a user I typically ignore the output, but if there's something wrong (say even internet connection) I can spot it just diving into the output, not the actual commands.

OneAgent installer one-liners don't use -q either.

**Review by dt-be — DISMISSED** [review]

**Review by dt-be — APPROVED** [review]

**Review by mmajcher — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ REQUIRE_VALID_CERTIFICATE=false
 Download the script and deploy the infrastructure:
 
 ```
-DT_AWS_LOG_FORWARDER=https://dynatraceawslogforwarder.s3.amazonaws.com && wget $DT_AWS_LOG_FORWARDER/preview/dynatrace-aws-logs.sh -O dynatrace-aws-logs.sh && chmod +x ./dynatrace-aws-logs.sh && wget $DT_AWS_LOG_FORWARDER/preview/lambda-aws-logs.zip -O lambda-aws-logs.zip && wget $DT_AWS_LOG_FORWARDER/preview/template-aws-logs.yaml -O template-aws-logs.yaml && ./dynatrace-aws-logs.sh deploy --target-url $TARGET_URL --target-api-token $TARGET_API_TOKEN --require-valid-certificate $REQUIRE_VALID_CERTIFICATE
+wget https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases/latest/download/dynatrace-aws-log-forwarder.zip && unzip -q dynatrace-aws-log-forwarder.zip && cd dynatrace-aws-log-forwarder && ./dynatrace-aws-logs.sh deploy --target-url $TARGET_URL --target-api-token $TARGET_API_TOKEN --require-valid-certificate $REQUIRE_VALID_CERTIFICATE
```
Comment (dt-be) [inline README.md:34]: && cd dynatrace-aws-log-forwarder

Why...
- [ ] One-liner that silently changes current dir is no go for me, let's get rid of this.

Comment (wojtasskorcz) [reply README.md:34]: Yeah, I also don't like it too much but `dynatrace-aws-logs.sh` has hardcoded CF template and Lambda zip paths which it expects in the working dir. Do you know a better way to approach this?

Comment (dt-be) [reply README.md:34]: Here's a proposal:
https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/pull/12/files

**Pull request merged**